### PR TITLE
Unapologetic Nigerian Prince Buffs

### PR DESCRIPTION
**Pull request by LordFowl**
Adds a few more spam texts to the pda spam event, including a more _procedural_ B-Movie generator.

### CODE REVIEW

**Comment by skull132** [comment]
Make spam into a JSON file when.

File: code/modules/events/money_spam.dm [hunk 1]
```diff
@@ -40,55 +40,118 @@
...
 					"Due to my lack of agents I require an off-world financial account to immediately deposit the sum of 1 POINT FIVE MILLION credits.",\
 					"Greetings sir, I regretfully to inform you that as I lay dying here due to my lack ofheirs I have chosen you to recieve the full sum of my lifetime savings of 1.5 billion credits")
+
 				if(6)
-					sender = pick("[current_map.company_name] Morale Divison","Feeling Lonely?","Bored?","www.wetskrell.nt")
+					var/pornography = pick("www.wetskrell.nt",\
+					"www.badkitty.nt",\
+					"www.scalesnstuff.nt",\
+					"www.spacelube.nt",\
+					"www.hivenetmingle.nt",)
```
Comment (Lohikar) [inline code/modules/events/money_spam.dm]: Comma.

File: code/modules/events/money_spam.dm [hunk 2]
```diff
@@ -40,55 +40,118 @@
...
+				if(8)
+					sender = pick("[pick(first_names_female,first_names_male)][pick(last_names)]@[current_map.company_name].[pick("ru","ck","tj","ur","nt")]",\
+					"Cindy Kate messenging",\
+					"MirandaTrasen@NanoTrasen.nt")
+					message = pick("We know who you are. Expect us.",\
+					"Hi [P.owner]! Remember me from high-school? Well I have this great business deal proposition....",\
+					"If you don't forward this message to 8 more crewmembers your head will die in their sleep.",\
+					"If you break the chain you will have bad luck for the rest of you're miserable life. Forward to 10 more crewmembers.",\
+					"Forward this email to 3.14 more crewmembers and recieve 10,000 cR!",\
+					"[P.owner]! I come from the future to give warning! The code has become self-referential! Run while you still can!")
```
Comment (Lohikar) [inline code/modules/events/money_spam.dm]: This one seems a little iffy (muh immulsions).

File: code/modules/events/money_spam.dm [hunk 3]
```diff
@@ -40,55 +40,118 @@
...
+					sender = pick("QuickDatingSystem",\
+					"Find your russian bride",\
+					"Tajaran beauties are waiting",\
+					"Find your secret skrell crush",\
+					"Beautiful unathi brides",\
+					"Hyperrealistic pleasure-bots available",\
+					"Zo'ra broods in need of genetic material!")
 					message = pick("Your profile caught my attention and I wanted to write and say hello (QuickDating).",\
-					"If you will write to me on my email [pick(first_names_female)]@[pick(last_names)].[pick("ru","ck","tj","ur","nt")] I shall necessarily send you a photo (QuickDating).",\
+					"If you will write to me on my email [pick(first_names_female)]@[pick(last_names)]@[current_map.company_name].[pick("ru","ck","tj","ur","nt")] I shall necessarily send you a photo (QuickDating).",\
```
Comment (Pacmandevil) [inline code/modules/events/money_spam.dm]: pablo@rodregaz@nanotrasen.ru?

Comment (Arrow768) [reply code/modules/events/money_spam.dm]: It´s a spammer, what do you expect ?

**Pull request merged**